src/pdf.js

```diff
@@ -42,10 +42,8 @@ if (!PDFJS.workerSrc && typeof document !== 'undefined') {
   // workerSrc is not set -- using last script url to define default location
```

Comment:
Perhaps it would be a good idea to also add a `deprecated` warning for `PRODUCTION` mode?